src/main/scala/com/amazon/deequ/examples/KLLExample.scala

```diff
@@ -32,13 +32,13 @@ private[examples] object KLLExample extends App {
       Item(4, "Thingy D", "checkout https://thingd.ca", "low", 10),
       Item(5, "Thingy E", null, "high", 12))
 
-    val suggestionResult = ConstraintSuggestionRunner()
+    val profileResult = ColumnProfilerRunner()
       .onData(df)
-      .addConstraintRules(Rules.DEFAULT)
-      .setKLLParameters(KLLParameters(2, 0.64, 2))
+      .withKLLProfiling()
+      .setKLLParameters(Some(KLLParameters(2, 0.64, 2)))
       .run()
 
-    val columnProfiles = suggestionResult.columnProfiles
+    val columnProfiles = profileResult.profiles
 
     println("Observed statistics:")
```

Comment:
I think we can substitute those printing lines below with the single line `println(ColumnProfiles.toJson(columnProfiles.map { case (_, profile) => profile }.toSeq))`, but not sure you want that.